base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py

```diff
@@ -12,11 +12,18 @@
 table = widget.getTable()
 row = 1
 color = ColorFontUtil.getColorFromRGB(240,245,245)
+
 i = 0
 for i in range(20):
-    reading = PVUtil.getString(pvs[i]) 
+    reading = PVUtil.getString(pvs[i])
+    temperature = PVUtil.getString(pvs[i+20])
```

Comment:
Beware of magic numbers. Is this `20` the same as the `20` in `for i in range(20):`? I guess so, but it's not obvious. It's better to have.

```
offset_between_temperature_and_drift_pv_indexes = 20
...
```

It's maybe a tad verbose, but something along those lines rould remove confusion

Reply:
Removed magic numbers